### PR TITLE
feat(addon-commerce): rename `tui-card` to `tui-thumbnail-card`

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/demo/src/modules/app/app.routes.ts [hunk 1]
```diff
@@ -244,7 +244,8 @@ export const ROUTES: Routes = [
     {
         path: `components/card`,
         loadChildren: async () =>
-            (await import(`../components/card/card.module`)).ExampleTuiCardModule,
+            (await import(`../components/thumbnail-card/card.module`))
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/app.routes.ts:255]: `thumbnail-card.module.ts` ?

File: projects/demo/src/modules/components/thumbnail-card/card.module.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import {TuiThumbnailCardExample2} from './examples/2';
+import {TuiThumbnailCardExample3} from './examples/3';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        TuiThumbnailCardModule,
+        RouterModule,
+        TuiAddonDocModule,
+        RouterModule.forChild(tuiGenerateRoutes(ExampleTuiThumbnailCardComponent)),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/thumbnail-card/card.module.ts:18]: `RouterModule` x2

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/thumbnail-card/card.module.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/thumbnail-card/card.module.ts]: fixed

File: projects/demo/src/modules/app/app.routes.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/app/app.routes.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**